### DISCUSSION TITLE
Relay settings

### DISCUSSION THREAD

**Discussion by RichardFevrier** [rustdesk/rustdesk#4547] [Feature Request]
Hi, I would know if we could have a setting to control the relay usage (auto/enabled/disabled) from the remote and/or the client.
Thanks for your amazing work 🙏

**Comment by SlalomJohn** [comment]
IMHO - from client - no.
This settings on server side
Details - look my post [here](https://github.com/rustdesk/rustdesk/discussions/966#discussioncomment-6070702)

**Reply by RichardFevrier** [reply]
I have inverted the words "remote" and "client" to change the priority(feasibility) order.
But I don't see the link with your other post @SlalomJohn with this feature request.

**Reply by SlalomJohn** [reply]
Environment="ALWAYS_USE_RELAY=Y"
Environment="RELAY_SERVERS=your_server_ip_or_name_where_is_running_hhbr"
